docker/Dockerfile

```diff
@@ -35,6 +36,12 @@ ENV DOWNLOAD=${DOWNLOAD}
 
 MAINTAINER FIWARE IoTAgent Team. Telefónica I+D
 
+# IMPORTANT: For production environments use Docker Secrets to protect values of the sensitive ENV 
+# variables defined below, by adding _FILE to the name of the relevant variable.
+#
+# - IOTA_AUTH_USER, IOTA_AUTH_PASSWORD when using Keystone Security: 
```

Comment:
The `:` at the end of the sentence seems to mean that something is missing at the end or in a next line. Is that a typo? Or actually something is missing?

Reply:
Fixed 604b2b9 - That was an artefact left from me re-structuring and re-writing the comment to put the variables at the start of the line trying to avoid looking like it is a deliberately commented out ENV

Reply:
To keep things aligned, I understand the same change must be implemented for IOTA-UL in https://github.com/telefonicaid/iotagent-ul/pull/346

With regards to IOTA-JSON I think that https://github.com/telefonicaid/iotagent-json/blob/master/docker/Dockerfile#L21 should be also changed. I'll also do so and tell you about the PR when it gets ready.